### PR TITLE
docs: Figma Link Update

### PR DESCRIPTION
**Pull request by songstephen**
### Overview

Updated storybook pages to link to the most up-to-date Figma components.

### PR Checklist

- [X] Related to designs:
- [X] Related to JIRA ticket:
- [X] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@42.3.3-alpha.f8d3fc.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6102ded4eed7b1454ac36eb3--gamut-preview.netlify.app>https://6102ded4eed7b1454ac36eb3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6102ded4eed7b1454ac36eb3>Deploy Logs</a></p>

**Review by codecaaron — APPROVED** [review]
🥳

File: packages/styleguide/stories/Molecules/GlobalFooter.stories.mdx [hunk 1]
```diff
@@ -9,8 +9,7 @@ import { Canvas, Meta, Props, Story } from '@storybook/addon-docs/blocks';
     source: 'gamut-labs',
     status: 'volatile',
     subtitle: `Global footer to be displayed on the bottom of each page.`,
-    viewMode: 'canvas',
-    layout: 'fullscreen',
+    figmaId: 'ReGfRNillGABAj5SlITalN/%F0%9F%93%90-Gamut?node-id=735%3A4',
```
Comment (codecaaron) [inline packages/styleguide/stories/Molecules/GlobalFooter.stories.mdx:12]: Special characters: 
```suggestion
    figmaId: `ReGfRNillGABAj5SlITalN/%F0%9F%93%90-Gamut?node-id=735%3A4`,
```

Comment (songstephen) [reply packages/styleguide/stories/Molecules/GlobalFooter.stories.mdx:12]: Should all the `figmaId:` fields be using the ` or ' apostrophes? I see the subtitle in this file is using the special character but don't know what's best practice or consistent here.

Comment (codecaaron) [reply packages/styleguide/stories/Molecules/GlobalFooter.stories.mdx:12]: Oops - they don't need to be :).  Ideally they all would be single apos' but some characters do weird things to the formatting.

Comment (codecaaron) [reply packages/styleguide/stories/Molecules/GlobalFooter.stories.mdx:12]: @songstephen Woops I read the error wrong here - we need to delete this https://github.com/Codecademy/client-modules/pull/1928/files#diff-f88e0c6da21130b5e5da275ef17561203020cbb56a110359cfa16016df192f55L8 on the object.

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61045acc234cb00daf8189b0--gamut-preview.netlify.app>https://61045acc234cb00daf8189b0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61045acc234cb00daf8189b0>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@42.3.6-alpha.447eca.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6104677651be923173911e55--gamut-preview.netlify.app>https://6104677651be923173911e55--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6104677651be923173911e55>Deploy Logs</a></p>

**Pull request merged**